test/test_manifest.json

```diff
@@ -1191,6 +1191,12 @@
        "link": true,
        "type": "eq"
     },
+    {  "id": "Highlight.without.appearance.stream",
+       "file": "pdfs/Highlight.without.appearance.stream.pdf",
+       "md5": "a1f2811324fa1ff0c9f1778697413dad",
+       "rounds": 1,
+       "type": "eq"
```

Comment:
The new tests in this file _all_ need to have `"annotations": true` in order to render the annotations.